marklogic-data-hub-central/ui/src/pages/Run.tsx

```diff
@@ -373,6 +373,10 @@ const Run = (props) => {
     let checkStatus = (resolve, reject) => {
       let promise = fn();
       promise.then(function(response) {
+        if (!response.data) {
+          throw new Error("Empty response body received");
```

Comment:
We throw an error here if the data is empty so that it can be caught by the FE's retry mechanism.

Reply:
Does it show up in the browser's console log though? That would seem odd, since this isn't an error condition; the job document just hasn't been written yet.

Reply:
No it does not, its caught by the retry mechanism which only throws its console error after repeated polling attempts.